moped-database/README_seed_data.md

```diff
@@ -0,0 +1,39 @@
+### How to generate new seeds
+
+1. Navigate to `./moped-database`
+2. Start the hasura cluster
+3. Add / edit whatever data
+4. ❗️Warning: adding subprojects creates a circular reference that will cause seeds to fail
```

Comment:
Thank you for advancing our seed game. This is hard work that will affect the DX massively. 🙏